alignak_backend/app.py

```diff
@@ -1862,6 +1876,23 @@ def get_settings(prev_settings):
 app.on_pre_PATCH += pre_patch
 app.on_pre_DELETE += pre_delete
 app.on_insert_user += pre_user_post
+
+# Manage alias when insert
+app.on_insert_alignak += pre_post_alias
+app.on_insert_command += pre_post_alias
+app.on_insert_host += pre_post_alias
+app.on_insert_hostdependency += pre_post_alias
+app.on_insert_hostescalation += pre_post_alias
+app.on_insert_hostgroup += pre_post_alias
+app.on_insert_realm += pre_post_alias
+app.on_insert_service += pre_post_alias
+app.on_insert_servicedependency += pre_post_alias
+app.on_insert_serviceescalation += pre_post_alias
+app.on_insert_servicegroup += pre_post_alias
+app.on_insert_timeperiod += pre_post_alias
+app.on_insert_user += pre_post_alias
+app.on_insert_usergroup += pre_post_alias
+
```

Comment:
Why not having one global on_insert hook? In this case if any posted data is an alias field it is tested and modified. The code would be much more simple to maintain, no?